FirebaseAdmin/FirebaseAdmin/Auth/Providers/OidcProviderConfig.cs

```diff
@@ -63,13 +65,60 @@ internal OidcProviderConfig(Request request)
         /// </summary>
         public string Issuer { get; }
 
+        /// <summary>
+        /// Gets the Client Secret used to verify code based response types.
+        /// </summary>
+        public string ClientSecret { get; }
+
+        /// <summary>
+        /// Gets what response types are being used for this OIDC config.
+        /// </summary>
+        public ResponseTypeArgs ResponseType { get; }
```

Comment:
This doesn't follow the signatures proposed at go/gcip-oidc-provider-code-flow. I think we want separate boolean getters at the top-level (similar to the Java API).

```cs
bool IDTokenResponseType { get; }

bool CodeResponseType { get; }
```

Reply:
Done